contracts/network/interfaces/IPendingWithdrawals.sol

```diff
@@ -66,13 +66,14 @@ interface IPendingWithdrawals is IUpgradeable {
 
     /**
      * @dev cancels a withdrawal request
+     * returns the value of the position in units of the reserve token
```

Comment:
`returns the number of reserve tokens associated with the withdrawal request`

Reply:
done